package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@justeat/fozzie",
   "title": "Fozzie – Just Eat UI Web Framework",
   "description": "UI Web Framework for the Just Eat Global Platform",
-  "version": "0.51.3",
+  "version": "0.53.0",
```

Comment:
Is there a v52?  If not it's probably worth updating the changelog so that the last version was `v0.51.3` and this one is then `v0.52.0`

Reply:
the previous release was v52 but typed in the wrong number. Do we need to release a minor with a 52 version bump before this?

Reply:
I would yeh, just to keep the changelog correct with the releases on NPM, as there isn't a `v0.52.0` on there atm

Reply:
PR for you 😁 
https://github.com/justeat/fozzie/pull/95

Reply:
I've merged that one – if you can release if this one should be good to go then too